Vic2ToHoI4Tests/MapperTests/MergeRules/MergeRulesTests.cpp

```diff
@@ -0,0 +1,18 @@
+#include "Configuration.h"
+#include "Mappers/MergeRules/MergeRules.h"
+#include "Mappers/MergeRules/MergeRulesFactory.h"
+#include "gmock/gmock-matchers.h"
+#include "gtest/gtest.h"
+
+
+
+TEST(Mappers_MergeRules_MergeRulesTests, RulesCanBeImported)
+{
+	const auto mergeRules = Mappers::MergeRules::Factory().importMergeRules();
+
+	ASSERT_THAT(mergeRules->getRules(),
```

Comment:
probably reverse arguments. Unsure with ASSERT_THAT.

Reply:
ASSERT_THAT forces this order